README.md

```diff
@@ -2,47 +2,64 @@
 
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/db5eda4c50b24c008b2c19f1117c9cef)](https://www.codacy.com/gh/codacy/codacy-coverage-reporter-action/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=codacy/codacy-coverage-reporter-action&amp;utm_campaign=Badge_Grade)
 
-> 🤖 Automated multi-language coverage reporter for [Codacy](https://www.codacy.com/).  
-> For more information about Codacy see our [5 minutes quickstart](https://docs.codacy.com/getting-started/codacy-quickstart/).
+GitHub Action for [uploading coverage reports to Codacy](https://docs.codacy.com/coverage-reporter/#uploading-coverage) on all commits and pull requests, tracking code coverage, and returning code coverage issues.
 
-This action runs the [codacy-coverage-reporter](https://github.com/codacy/codacy-coverage-reporter) for all commits and pull requests with your Codacy configuration.
+[Codacy](https://www.codacy.com/) is an automated code review tool that makes it easy to ensure your team is writing high-quality code by analyzing more than 40 programming languages such as PHP, JavaScript, Python, Java, and Ruby. Codacy allows you to define your own quality rules, code patterns and quality settings you'd like to enforce to prevent issues on your codebase.
 
-## Usage
+<br/>
 
-Create a new workflow `.yml` file in the `.github/workflows/` directory.
+<a href="https://www.codacy.com" target="_blank"><img src="images/codacy-logo.svg" alt="Codacy" width="400"/></a>
 
-### .github/workflows/codacy-coverage-reporter.yml
+<br/>
 
-```yml
-name: codacy-coverage-reporter
+## Coverage using your project’s quality settings
```

Comment:
Here, we can make the title more straightforward (even if it loses some consistency with the ones from the Codacy Analysis CLI):

```suggestion
## Uploading coverage to Codacy
```

In practice, the GitHub Action knows nothing about the repository quality settings configured on Codacy. The use case for the GitHub Action is just to ensure that it uploads the relevant coverage reports for each commit pushed to the repository.